### ISSUE TITLE
Exception when just using Col

### ISSUE THREAD

**Issue by ericraio** [reactstrap/reactstrap#49]
I am getting this exception when I am just using the Col component.

I am using react 15.2.0-rc.1

`warning.js?8a56:44 Warning: Unknown prop `md` on <div> tag. Remove this prop from the element. For details, see https://fb.me/react-unknown-prop`

I think the fix would be to list out all the proptypes on line 68 in lib/Col.js

```javascript
var attributes = _objectWithoutProperties(props, ['className'].concat(Object.keys(propTypes)));
```

**Comment by eddywashere** [comment]
Thank you so much for reporting this. I'll update to explicitly list out props. I believe other components are affected by this as well.

**Comment by ericraio** [comment]
@eddywashere awesome! Looking forward to the update.

**Comment by eddywashere** [comment]
@ericraio based on first pass through docs/tests with React 15.2.1, this should be fixed with 1.6.0 https://github.com/reactstrap/reactstrap/pull/52
